tools/Dashboard/DevHome.Dashboard/ViewModels/WidgetViewModel.cs

```diff
@@ -114,63 +114,65 @@ private async Task RenderWidgetFrameworkElementAsync()
     {
         await Task.Run(async () =>
         {
-            var cardTemplate = await Widget.GetCardTemplateAsync();
-            var cardData = await Widget.GetCardDataAsync();
+        var cardTemplate = await Widget.GetCardTemplateAsync();

```

Comment:
Please fix/revert the indentation just so looking at the diff is easier. (control+KD in VS should do it)

Reply:
rookie mistake - fixing now